app/serializers/rest/instance_serializer.rb

```diff
@@ -24,6 +24,12 @@ def version
     Mastodon::Version.to_s
   end
 
+  def stats
+    { user_count:   User.confirmed.count,
```

Comment:
It would be better to use [`InstancePresenter`](https://github.com/tootsuite/mastodon/blob/09e86ef/app/presenters/instance_presenter.rb).